pkg/sql/sqlstats/insights/detector.go

```diff
@@ -170,3 +171,14 @@ func (d *latencyThresholdDetector) isSlow(s *Statement) bool {
 func isFailed(s *Statement) bool {
 	return s.Status == Statement_Failed
 }
+
+// ignoreStatement returns true if we don't want to analyze the statement.
+func ignoreStatement(s *Statement) bool {
+	ignore := []string{"SET "}
```

Comment:
Is it possible to init this outside of the function so we don't need to do it every time? Not sure if the compiler would just optimize this away though.